README.md

```diff
@@ -5,6 +5,34 @@
 The project is still work in progress.
 Is possible to generate a working `danger-kotlin` instance from the code that is currently on this repo, but is not ready to be distributed yet.
 
+### What it looks like today
+You can make a Dangerfile that looks through PR metadata, it's fully typed.
+
+```kotlin
+import com.danger.dangerkotlin.*
+import org.jetbrains.kotlin.script.util.*
+
+val danger = Danger(args)
+
+val allSourceFiles = danger.git.modifiedFiles + danger.git.createdFiles
+
+val changelogChanged = allSourceFiles.contains("CHANGELOG.md")
+val sourceChanges = allSourceFiles.firstOrNull { it.contains("src")  }
+val isTrivial = danger.github!!.pullRequest.title.contains("#trivial")
+
+if (!isTrivial && !changelogChanged && sourceChanges != null) {
+    warn("Any changes to library code should be reflected in the Changelog.\n\nPlease consider adding a note there and adhere to the [Changelog Guidelines](https://github.com/Moya/contributors/blob/master/Changelog%20Guidelines.md).")
+}
+
+if (danger.git.createdFiles.size + danger.git.modifiedFiles.size - danger.git.deletedFiles.size > 10) {
+    warn("Big PR, try to keep changes smaller if you can")
+}
+
+if (danger.github!!.pullRequest.title.contains("WIP" ,false)) {
```

Comment:
`.github!!` this something you can do at declaration-ish point? (like how in Swift/TS it pretends to not be be nullable)

Reply:
I didn't find a way to force unwrap on the type declaration, and `Moshi` (the library that we are using) requests an optional for this.
A valid alternative could be make this private and expose a getter that force unwraps it

Reply:
I've opened https://github.com/danger/kotlin/pull/42, let's see if fixes this

Reply:
approved even if i don't like `_varName` 🙈